src/test/java/com/okta/tools/aws/settings/CredentialsTest.java

```diff
@@ -130,34 +85,7 @@ void addOrUpdateProfileToExistingProfile() throws IOException {
 
         final String updatedPrefix = "updated_";
         final String expected = existingCredentials + "\n\n"
-                + "[" + roleName + "_source]\n"
-                + Credentials.ACCES_KEY_ID + " = " + updatedPrefix + accessKey + "\n"
-                + Credentials.SECRET_ACCESS_KEY + " = " + updatedPrefix + secretKey + "\n"
-                + Credentials.SESSION_TOKEN + " = " + updatedPrefix + sessionToken;
-
-        credentials.addOrUpdateProfile(roleName, updatedPrefix + accessKey, updatedPrefix + secretKey, updatedPrefix + sessionToken);
-        credentials.save(credentialsWriter);
-
-        String given = org.apache.commons.lang.StringUtils.remove(credentialsWriter.toString().trim(), '\r');
-
-        assertEquals(expected, given);
-    }
-
-    /*
-     * Test updating an existing profile with custom suffix.
-     *
-     * Leaves the entry with the previous suffix in place by design.
-     */
-    @Test
-    void addOrUpdateProfileToExistingProfileCustomSuffix() throws IOException {
-        final StringReader credentialsReader = new StringReader(existingCredentials + "\n\n" + manualRole);
-        final StringWriter credentialsWriter = new StringWriter();
-        final Credentials credentials = new Credentials(credentialsReader, environmentWithCustomSuffix);
-
-        final String updatedPrefix = "updated_";
-        final String expected = existingCredentials + "\n\n"
-                + manualRole + "\n\n"
-                + "[" + roleName + "_custom]\n"
```

Comment:
:fire: suffix customization was a workaround for the workaround causing problems for people using this with downstream tools. Removing the writes to ~/.aws/config makes testing suffix customization unnecessary.